### PR TITLE
update TB QC comment

### PR DESCRIPTION
**Pull request by fevac**
## Description

https://github.com/Clinical-Genomics/MTP-TOMTE/issues/24

### Changed

- Add information to be shown in TB comment for QC checks (sample id, sample name and threshold)

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b update_qc_comment -a
    ```

### How to test

- [x] `trailblazer set-status --status qc tendergoose`
- [x] `cg workflow rnafusion metrics-deliver tendergoose`

### Expected test outcome

- [x] Check that comment is properly added: 
image

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Comment by fevac** [comment]
Tests are passing

File: cg/meta/workflow/nf_analysis.py [hunk 1]
```diff
@@ -745,7 +745,11 @@ def validate_qc_metrics(self, case_id: str, dry_run: bool = False) -> None:
         except MetricsQCError as error:
             LOG.error(f"QC metrics failed for {case_id}, with: {error}")
             self.trailblazer_api.set_analysis_status(case_id=case_id, status=AnalysisStatus.FAILED)
-            self.trailblazer_api.add_comment(case_id=case_id, comment=str(error))
+            for sample in self.status_db.get_samples_by_case_id(case_id=case_id):
```
Comment (seallard) [inline cg/meta/workflow/nf_analysis.py:748]: For readability, consider splitting this line like
```python
samples = self.status_db.get_samples_by_case_id(case_id):
for sample in samples:
    ...
```

**Review by seallard — APPROVED** [review]
👍

**Comment by fevac** [comment]
deployed

image

image

**Pull request merged**